### PR TITLE
Update data.js

### PR DESCRIPTION
**Pull request by Tanya-Jain**
Addition of data in 'About' section and events previously organised

### CODE REVIEW

**Comment by aniketmaithani** [comment]
@Tanya-Jain please 
- `update your forked repo` 
- `rebase the branch and your changes`
- For more info refer to README.md file.

File: js/data.js [hunk 1]
```diff
@@ -43,8 +47,13 @@ services[5] = "Already know something? Teach it to others! That's how we work!";
...
+events[2] = {title: "Weekly Sync Up Call", cta: "https://bluejeans.com/538914591", ctaText: "Join Now", description: "Weekly SYnc Up Call", date: "9 November 2016", startTime: "9:00 PM IST", endTime: "10:00 PM IST", location: "BlueJeans", eventOD: 0};
+events[3] = {title: "Web Scraping Using Python", cta: "#", ctaText: "Register Now", description: "Websites such as e-commerce, social networks, real estate, etc., display data that can only be viewed using a web browser, and lack the functionality to save a copy of this data to a local file on computer. Web Scraping technique automates this process. Its software interacts with websites as a web browser does, prevents manually copying the data from websites, and performs the task within a fraction of time.", date: "11 November 2016", startTime: "2:30 PM IST", endTime: "3:30 PM IST", location: "E3-311, Amity University, Sec-125", eventOD: 0};
+events[4] = {title: "Weekly Sync Up Call", cta: "https://bluejeans.com/538914591", ctaText: "Join Now", description: "Weekly Sync Up Call", date: "9 December 2016", startTime: "9:00 PM IST", endTime: "10:00 PM IST", location: "BlueJeans", eventOD: 0};
```
Comment (bhanuvrat) [inline js/data.js]: lines should not be more than 120 characters.

Comment (aniketmaithani) [inline js/data.js]: + 1 @bhanuvrat

**Comment by aniketmaithani** [comment]
@Tanya-Jain I am closing this as of now. Feel free to rebase and correct the conflict and then send the PR.

**Pull request closed without merging**